packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -2,18 +2,67 @@
 const expect = require('expect');
 const { accessibility } = require('../commands/validates');
 const { toBeAccessible } = require('../commands/expect');
+const { getViewports, describeViewports, setViewport } = require('../commands/viewport-helpers');
+
+const hideInputCaret = require('../commands/hide-input-caret');
 
 class TerraService {
+  constructor(options = {}) {
+    this.formFactor = options.formFactor;
+  }
+
   /**
    * Service hook executed prior to test execution.
    * Initializes the Terra Service's custom commands.
    */
-  before() {
+  before(capabilities) {
+    // Add the Jest expect module the use the Jest matchers.
     global.expect = expect;
     global.expect.extend({ toBeAccessible });
 
-    global.Terra = {};
-    global.Terra.validates = { accessibility };
+    // Add a Terra global with access to Mocha-Chai test helpers.
+    global.Terra = {
+      validates: { accessibility },
+
+      // viewports provides access Terra's list of test viewports.
+      viewports: getViewports,
+
+      // describeViewports provides a custom describe block for looping test viewports.
+      describeViewports,
```

Comment:
If you bind this, can we access the formFactor in the describeViewports command like `this.formFactor`?

```js
describeViewports: describeViewports.bind(this)
```